packages/lesswrong/components/comments/CommentsItem/CommentBody.tsx

```diff
@@ -37,6 +53,99 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
+export type PotentialCommentsData = {
+  potentialComments: {content: string|null, doppelId: string|null}[],
+  activePotentialComment: number,
+  setActivePotentialComment: React.Dispatch<SetStateAction<number>>,
+  doppelCommentVote: React.ReactNode,
+}
+
+
+const Content = ({content, contentType, commentBodyRef, bodyClasses, comment, highlights}:
+  { content: string,
+    contentType: ContentStyleType,
+    commentBodyRef?: React.RefObject<ContentItemBody>|null,
+    bodyClasses: string,
+    comment: CommentsList,
+    highlights?: Record<string,ContentReplacedSubstringComponentInfo>,
+  }) => {
+    const { ContentStyles, ContentItemBody } = Components;
+    const classes = useStyles(styles, "CommentBody");
+    return <ContentStyles contentType={contentType} className={classes.root}>
+      <ContentItemBody
+        ref={commentBodyRef ?? undefined}
+        className={bodyClasses}
+        dangerouslySetInnerHTML={{__html: content}}
+        description={`comment ${comment._id}`}
+        nofollow={(comment.user?.karma || 0) < nofollowKarmaThreshold.get()}
+        replacedSubstrings={highlights}
+      />
+    </ContentStyles>
+};
+
+const TabContent = ({doppelCommentVote, potentialComments, commentBodyRef, comment, bodyClasses, innerHtml, contentType, highlights}: PotentialCommentsData & {
+  comment: CommentsList,
+  contentType: ContentStyleType,
+  bodyClasses: string,
+  commentBodyRef?: React.RefObject<ContentItemBody>|null,
+  innerHtml: string,
+  highlights?: Record<string,ContentReplacedSubstringComponentInfo>,
+}) => {
+  const [activePotentialComment, setActivePotentialComment] = React.useState(0);
+  const maxPotentialCommentLength = Math.max(...potentialComments.map(p => p.content?.length ?? 0));
+  const classes = useStyles(styles, "CommentBody");
+  return <>
+    <div className={classes.doppelQuizIntro}>
+      <h3 className={classes.doppelQuizHeader}>Which of these 'comments' is the real one, and which generated by LM?</h3>
+      {maxPotentialCommentLength > 3_000 ? doppelCommentVote : null}
+    </div>
+    <Tabs
+      value={activePotentialComment}
+      onChange={(_event, newValue) => setActivePotentialComment(newValue)}
```

Comment:
nit: no need to prefix `event` with an underscore, if you want to skip using the first arg just alias it as `_`